integration-tests/sanity-functional-protractor/helpers/random-data.js

```diff
@@ -0,0 +1,15 @@
+ 'use strict';
+
+ module.exports = (function pdjWait() {
```

Comment:
Hi, sometimes you using function names in module.exposts, sometimes declare anonymous function, please clean up all this declarations or use some naming conversion.

Reply:
done